dll/win32/uxtheme/CMakeLists.txt

```
@@ -9,6 +9,7 @@ list(APPEND SOURCE
     main.c
     metric.c
     msstyles.c
+    png.cpp
```

```suggestion
    pngsup.c
```
i.e. PNG support or supplemental. Also every other code file here is "C".

---

Suggesting this:
```cmake
if(DLL_EXPORT_VERSION GREATER_EQUAL 0x600)
    list(APPEND SOURCE
        pngsup.cpp)
endif()
```